modules.js

```diff
@@ -214,42 +322,32 @@ const createModule = async (
           return false;
         }
         return true;
+      case 'global-samplejsr.css':
+      case 'stories':
+      case 'tests':
+        if (getInternalVersion) {
+          return true;
+        }
+        return false;
       default:
         return true;
     }
   };
 
-  const folderName =
-    !name || name.endsWith('.module') ? name : `${name}.module`;
-  const destPath = path.join(dest, folderName);
-  if (!options.allowExistingDir && fs.existsSync(destPath)) {
-    logger.error(
-      i18n(`${i18nKey}.errors.pathExists`, {
-        path: destPath,
-      })
-    );
-    return;
-  } else {
-    logger.log(
-      i18n(`${i18nKey}.creatingPath`, {
-        path: destPath,
-      })
-    );
-    fs.ensureDirSync(destPath);
-  }
-
-  logger.log(
-    i18n(`${i18nKey}.creatingModule`, {
-      path: destPath,
-    })
-  );
+  // Download gitHub contents to the dest directory
+  const sampleAssetPath = !isReactModule ? 'Sample.module' : 'SampleJSR';
```

Comment:
```
const sampleAssetPath = isReactModule ? 'SampleReactModule : 'Sample.module';
```

Reply:
Updated the [sample asset](https://github.com/HubSpot/cms-sample-assets/pull/6) to reflect this -- will update code here as well